packages/sockethub/package.json

```diff
@@ -5,7 +5,8 @@
   "private": false,
   "author": "Nick Jennings <nick@silverbucket.net>",
   "license": "LGPL-3.0+",
-  "main": "./bin/sockethub",
+  "main": "yarn run utart",
```

Comment:
I don't think that's what "main" is for. It usually points to a JS file to use with programs like nodemon for example.

Reply:
The `main` property should point to the file that contains the export you want to be given when doing a `require('sockethub')`. See https://docs.npmjs.com/files/package.json#main

Reply:
Right, makes sense now :) thanks - I knew this but somewhere along the line conflated it with the bin property. Anyway, it's fixed now.